modules/hub/README.rst

```diff
@@ -26,6 +26,14 @@ Running Tests
 
    $ ./migrate-and-test.sh
 
+Creating Migrations
```

Comment:
add here that you have to add it to the database module as well. we could make a precommit hook for that?

Reply:
@bohendo how do you think we should handle this? We should probably have the CI process pull in migrations from the hub module into the db module.